spec/lib/api/2.0/nodes-spec.js

```diff
@@ -722,5 +724,14 @@ describe('Http.Api.Nodes', function () {
                 });
         });
 
+        it('should call masterDelTagById', function() {
+            return helper.request().delete('/api/2.0/nodes/tags/name')
```

Comment:
I'd suggest having nodesApi.masterDelTagById resolve to an array of mock Ids and then expect delete response body to match that array.